README.md

```diff
@@ -40,49 +40,46 @@ host):
 
 1. Copy `docker-inasafe/realtime-rest.env.template` to `docker-inasafe/realtime-rest.env` and edit if you want to change the Realtime REST server url.
 
-1.  For the four Docker images that will be built (listed above), if you want to use apt-cacher-ng then follow the instructions in each `71-apt-cacher-ng.template` file and in each `Dockerfile` file uncomment the `#ADD 71-apt-cacher-ng /etc/apt/apt.conf.d/71-apt-cacher-ng` line.
-```
+1.  For the four Docker images that will be built (listed above), if you want to use apt-cacher-ng then follow the instructions in each `71-apt-cacher-ng.template` file and in each `Dockerfile` file uncomment the `#ADD 71-apt-cacher-ng /etc/apt/apt.conf.d/71-apt-cacher-ng` line, then
 
-then
+    ```
+    git clone git://github.com/inasafe/docker-realtime-orchestration.git
+    cd docker-realtime-orchestration
+    make
+    make checkout __production environment__ OR
```

Comment:
While you're at it, can you fix this? it should be just make checkout. The old command is not relevant anymore